README.md

```diff
@@ -37,23 +37,23 @@ Example bot
 import { Bot, Context } from "https://deno.land/x/grammy/mod.ts";
 import { I18n, I18nFlavor } from "https://deno.land/x/grammy_i18n/mod.ts";
 
+// For proper typings and auto-completions in IDEs,
+// customise the `Context` using `I18nFlavor`.
+type MyContext = Context & I18nFlavor;
+
 // Create a new I18n instance.
-const i18n = new I18n({
+const i18n = new I18n<MyContext>({
   defaultLocale: "en",
   directory: "locales",
 });
 
-// For proper typings and auto-completions in IDEs,
-// extend the `Context` using `I18nFlavor`.
-type MyContext = Context & I18nFlavor;
-
 // Create a bot as usual, but use the modified Context type.
 const bot = new Bot<MyContext>(""); // <- Put your bot token here
 
 // Tell the bot to use the middleware from the instance.
 // Remember to register this middleware before registering
 // other middlewares.
```

Comment:
Other middlewares? User middlewares maybe? Some middlewares (like session) must be installed before

Reply:
Maybe “before registering your handlers”?

Reply:
Yep, sounds good

Reply:
Also "Tell the bot to use the middleware from the instance." sounds too specific, doesn't it?

Reply:
Yes, I agree. It would be enough to say that the i18n plugin gets installed, what do you think?

Reply:
Just remove the first line

Reply:
Heh, yes that works!